packages/exceptions/src/InvalidArgumentException.ts

```diff
@@ -1,4 +1,4 @@
-import { RuntimeException } from './RuntimeException';
+import { RuntimeException } from './base/RuntimeException';
```

Comment:
Please put `Exception` and `RuntimeException` to root dir

Reply:
Why? Exceptions package already a complex one and will grow going forward. It's generally a bad practice to put everything in root - especially base classes.